benchmarks/util/gogoproto_generator.h

```diff
@@ -0,0 +1,98 @@
+// TODO: Insert description here. (generated by yilunchong)
```

Comment:
Merge this file with protoc-gen-gogoproto.cc

Reply:
Done